### DISCUSSION TITLE
External switches at certain times SC500W V2

### DISCUSSION THREAD

**Discussion by MMeinhardt1** [arendst/Tasmota#14816] [Q&A] [answered]
Hello,

I use LoraTap SC500W V2 to control my shutters and would like to know if I can disable the inputs of the external switches at certain times? The background is that in the future these will also be used in the children's rooms and then I would like to be able to control it at the certain times only by cell phone or tablet.
How can I solve this? For control I use ioBroker with the Sonoff adapter.

Thanks a lot
Michael

**Comment by barbudor** [comment, marked as answer]
You probably have configured buttons usage wiht commands `ShutterButton`
You can use the Timer function of Tasmota to execute a rule at a given time.
You need to set 2 Timer, 2 matching Rule trigger and in htose rules have one to deconfigure the buttons and 1 to reconfigure it back
More :
- ShutterButton: https://tasmota.github.io/docs/Blinds-and-Shutters/#button-control
- Timers: https://tasmota.github.io/docs/Timers/ and web configuration from Main Menu => Configuration => Timers
- Rules with Timer: https://tasmota.github.io/docs/Rules/#execute-several-commands-when-a-timer-expires
